web/modules/custom/dbcdk_community/src/Form/ProfileEditForm.php

```diff
@@ -24,79 +24,50 @@
 class ProfileEditForm extends FormBase implements ContainerInjectionInterface {
 
   /**
-   * The DBCDK Community Service Profile API.
+   * The profile repository to use.
    *
-   * @var \DBCDK\CommunityServices\Api\ProfileApi $profileApi
+   * @var \Drupal\dbcdk_community\Profile\ProfileRepository $profileRepository
```

Comment:
The ProfileRepository is declared at line 18, but you use the full name here - at line 36 it looks like you removed the namespace of the object `@var Profile $profile` - It is without any functional effect but my OCD thinks it would be nice if it was consistent :smile: